app/src/main/res/layout/statistics_frag.xml

```diff
@@ -17,63 +17,64 @@
 
     <data>
 
-        <import type="android.view.View" />
+        <import type="android.view.View"/>
 
         <variable
             name="stats"
-            type="com.example.android.architecture.blueprints.todoapp.statistics.StatisticsViewModel" />
+            type="com.example.android.architecture.blueprints.todoapp.statistics.StatisticsViewModel"/>
     </data>
 
-    <LinearLayout
+    <com.example.android.architecture.blueprints.todoapp.ScrollChildSwipeRefreshLayout
+        android:id="@+id/refresh_layout"
```

Comment:
nit: for a future change I want to standardize whether these ids are camelcase or _ --- what's the correct? There are examples of both in these layouts

Reply:
Sounds good for a future change yes